src/main/kotlin/racing/domain/Car.kt

```diff
@@ -1,22 +1,31 @@
 package racing.domain
 
-private const val DEFAULT_POSITION = 0
-private const val MAX_NAME_LENGTH = 5
-
 internal class Car(name: String = "", position: Int = DEFAULT_POSITION) {
-    init {
-        require(name.trim().length <= MAX_NAME_LENGTH) {
-            "The maximum name length is 5"
-        }
-    }
 
-    val name = name.trim()
+    val name: String = name.trim()
     var position: Int = position
         private set
 
-    fun tryMove(movable: Movable) {
+    init {
+        require(this.name.length <= MAX_NAME_LENGTH) {
+            "The maximum name length is $MAX_NAME_LENGTH"
+        }
+    }
+
+    fun tryMove(movable: Movable): Car {
         if (movable.isMovable()) {
             this.position++
```

Comment:
`Car`를 VALUE OBJECT로 관리하면 어떨까요?

```suggestion
            return Car(this.name, this.position + 1)
```